docs/asyncapi.yaml

```diff
@@ -0,0 +1,92 @@
+asyncapi: 3.0.0
+info:
+  title: FCP-FD-COMMS
+  version: 1.0.0
+  description: Receive a communication message and route to appropriate channels
+
+channels:
+  sourceSystem-fcp-fd-comms:
+    address: sourceSystem-fcp-fd-comms
+    description: |
+      Accepts POST requests for communication notifications.
+    messages:
+      subscribe.message.0:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/sourceSystem-fcp-fd-comms'
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/NotificationPayload'
+
+  schemas:
+    NotificationPayload:
+      type: object
+      properties:
+        CRN:
+          type: number
+          description: Customer Reference Number
+          minimum: 105000000
+          maximum: 999999999
+          example: 123456789
+        SBI:
+          type: number
+          description: Single Business Identifier
+          minimum: 105000000
+          maximum: 999999999
+          example: 123456789
+        sourceSystem:
+          type: string
+          description: The name of the source system that originated the message
+          example: "AHWP"
+        notifyTemplateId:
+          type: string
+          description: Notification template identifier
+          example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
+        commsType:
+          type: string
+          description: Type of communication (e.g., email, SMS)
+          example: "email"
+        data:
```

Comment:
I think `personalisation` could be brought up a level and this `data` item dropped.